src/python/bot/tasks/fuzz_task.py

```diff
@@ -1536,3 +1540,103 @@ def execute_task(fuzzer_name, job_type):
   del testcase_file_paths
   del testcases_metadata
   utils.python_gc()
+
+
+def fuchsia_qemu_setup():
+  """ Sets up and runs a QEMU VM in the background.
+  Does not block the calling process.
+  Fuchsia fuzzers assume a QEMU VM is running; call this routine prior to
+  beginning Fuchsia fuzzing tasks.
+  This initialization routine assumes that the GCS bucket contains the
+  standard Fuchsia SDK, as well as:
+  * /qemu-for-fuchsia/*
+  * /.ssh/* """
+  fuchsia_bucket_name = 'fuchsia_on_clusterfuzz_resources_v1'
+
+  resources_path = os.path.join(os.getcwd(), fuchsia_bucket_name)
+
+  shell.create_directory_if_needed(resources_path)
+
+  fuchsia_resources_url = environment.get_value('FUCHSIA_RESOURCES_URL')
+  if fuchsia_resources_url is None:
+    raise fuchsia.common.FuchsiaConfigError(
+        'Could not find path for remote'
+        'Fuchsia resources bucket (FUCHSIA_RESOURCES_URL')
+
+  gsutil_command_arguments = ['cp', '-r', fuchsia_resources_url, resources_path]
```

Comment:
@oliverchang What do you think of invoking `gsutil` directly instead of using `gsutil.py`? I'm inclined to accept it.

Reply:
We should be consistent with our use of GSUtilRunner here. It's also needed for local GCS mocking, so please use that instead :)